src/chrome/content/rules/stadtwiki.net.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+	Invalid certificate:
+		www.stadtwiki.net
+		allmende.stadtwiki.net
+		ettlingen.stadtwiki.net
+		tr.ka.stadtwiki.net
+
+	Redirect loop:
```

Comment:
Please double check, I think this has been fixed.

Reply:
Thanks, added them as targets.